pkg/build/strategies/strategies.go

```diff
@@ -37,8 +38,12 @@ func GetBuilderImage(config *api.Config) (*docker.PullResult, error) {
 	var image *dockerclient.Image
 	if config.ForcePull {
 		image, err = d.PullImage(config.BuilderImage)
+		if err != nil {
+			glog.Warningf("An error occurred when pulling %s: %v. Falling back to use local instead.", config.BuilderImage, err)
```

Comment:
Instead of "falling back to use local image" this should say "attempting to use local image" since the d.CheckImage could then also fail if the image doesn't exist locally at all.